ai_agents/openai_agent_researcher/workflow.py

```diff
@@ -0,0 +1,72 @@
+"""Common code to chat with the agent."""
+
+from os import getenv
+
+import data
+
+from ai import plan
+from ai import report
+from ai import search
+import slack
+from slack import ask
+from slack import send
```

Comment:
It's a bad practice to import specific functions, and it's much more readable to see where they're coming from when they're being called.

There are exceptions to that, especially with stdlib and 3rd-party imports, but not this.

```suggestion
import ai
import data
import slack
```

Reply:
fixed